.github/workflows/connector_teams_review_requirements.yml

```diff
@@ -19,7 +19,7 @@ jobs:
     name: "Check if a review is required from Connector teams"
     runs-on: ubuntu-latest
 
-    if: ${{ github.event.pull_request.head.repo.fork == false && github.event.pull_request.draft == false }}
+    if: ${{ github.event.pull_request.draft == false }}
```

Comment:
This was changed to skip running on forks, because forks didn't have access to do the check.

Reply:
Thanks, I got that - I misinterpreted Augustin's suggestion on your PR and thought that the GITHUB_TOKEN had access (or in this case that the contributor's token would have access). I still need this to run on forks, but i'll try what Augustin said about running it as a community_ci job, since using the master version should be no issue